app/retail/templates/shared/footer_scripts.html

```diff
@@ -15,35 +15,34 @@
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 {% endcomment %}
-{% load static %}
+{% load static compress %}
 <script type="text/javascript">
   var static_url = "{{ STATIC_URL }}";
   var media_url = "{{ MEDIA_URL }}";
   document.contxt = {{ json_context|safe }};
 </script>
-<script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
-<script src="{% static "v2/js/lib/jquery.js" %}"></script>
 {% if user.is_authenticated and env == 'prod' %}
-  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
-{% elif user.is_authenticated %}
-  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vue"></script> 
```

Comment:
Why the prod only check here?

Reply:
Production js https://cdn.jsdelivr.net/npm/vue
Dev + debugger js https://cdn.jsdelivr.net/npm/vue/dist/vue.js
we should keep https://cdn.jsdelivr.net/npm/vue/dist/vue.js for development or I not going to be able to debug vue when working on it.

Reply:
oh actually are diff codes the production vue doesn't have the debugger part that give us the vue errors so 
dist/vue.js > debugger tools
npm/vue > production code without debugger

Reply:
also shouldn't this change now to app/assets/v2/js/lib/vue.js ?